tools/ts/build.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+require("ts-node").register({
+  compilerOptions: {
+    strict: true,
+    target: "esnext"
+  },
+  skipProject: true
+});
+
+require("./build_lib");
```

Comment:
The name of this file and directory are too generic `tools/ts/build.js`. Is there something more descriptive we could use?

Maybe `tools/ts_declaration_builder/main.js`